commands/sms_commands/sms_recipet_get.js

```diff
@@ -0,0 +1,42 @@
+const cs = require('../../modules/clicksend.js')
+
+/**
+ ** Exports for the command line processor
+ **/
+
+exports.command = 'receipt';
+exports.desc = "Get a specific SMS Receipt";
+
+exports.builder = function (yargs) {
+    yargs
+        .option('messageId', {
+            describe: 'The message ID(s) for which you would like a receipt',
+            type: 'array',
+            requiresArg: true,
+            demandOption: true,
+            group: 'SMS Receipt Options:'
+        })
+}
+
+exports.handler = async function (yargs) {
+
+    var result = new Array();
```

Comment:
I've noticed yo've been using var and const at the same time, best practice is to just replace all `var` with either `const` or `let`. Since var are globally scoped, safer to do block scoping.